core/src/main/java/org/commonjava/maven/ext/manip/state/DependencyState.java

```diff
@@ -96,7 +101,13 @@ public DependencyState( final Properties userProps )
         ignoreSuffix = Boolean.valueOf( userProps.getProperty( STRICT_ALIGNMENT_IGNORE_SUFFIX, "false" ) );
         failOnStrictViolation = Boolean.valueOf( userProps.getProperty( STRICT_VIOLATION_FAILS, "false" ) );
         remoteBOMdepMgmt = IdUtils.parseGAVs( userProps.getProperty( DEPENDENCY_MANAGEMENT_POM_PROPERTY ) );
+
         dependencyExclusions = PropertiesUtils.getPropertiesByPrefix( userProps, DEPENDENCY_EXCLUSION_PREFIX );
+        // If dependencyExclusions have not been set via exclusion prefix attempt to set it via the override alias prefix.
+        if ( dependencyExclusions.isEmpty())
+        {
+            dependencyExclusions = PropertiesUtils.getPropertiesByPrefix( userProps, DEPENDENCY_OVERRIDE_PREFIX );
```

Comment:
@djcoleman  That is a good point. Currently on the command line, later overrides earlier. However we can't determine the order between `-DdepEx1 -DdepOver1 -DdepEx2 -DdepOver4` so another option is to merge and flag an error if there is a conflict.